### DISCUSSION TITLE
FATES software meeting google group

### DISCUSSION THREAD

**Discussion by glemieux** [NGEET/fates#906] [Ideas]
After [today's meeting](https://github.com/NGEET/fates/wiki/2022-Software-Engineering-meeting-notes#september-19-2022) @JessicaNeedham @mpaiao and I talked about having an email distribution list that is specific to the individuals who regularly attend the FATES software meetings.  This would provide us a way to easily reference the list and make sure we include all individuals.  Additionally, we could use it as a way to announce agenda updates or meeting cancellations.  How's that sound @rgknox @rosiealice @ckoven?

**Comment by rosiealice** [comment]
Good idea. Would make rapidly iterating on things a little easier. 
I wonder if anyone else might have time to volunteer on this? Perhaps we could bring that up at a wider fates meeting. I wouldn't want to leave someone out who was motivated to work on it...

**Reply by rosiealice** [reply]
I guess this is now 'closed' ?

**Reply by glemieux** [reply]
Yep, that's a good point.  I'll lock to discussion so that it can't be further added to.
